### PR TITLE
Sufyan/sustain 12

### PR DESCRIPTION
**Pull request by skhan943**
# JIRA Ticket Link
https://skaarkzo-capstone.atlassian.net/browse/SUSTAIN-12

# Summary of Changes
Set up project structure, installed basic packages, created requirements file, and set up fast api with hello world route.

### CODE REVIEW

**Review by zaidokal — DISMISSED** [review]
Looks Good!

**Comment by rakhshankhan** [comment]
Do we wanna add some more middleware implementation? Such as CORS and also to specify the port on which this service is gonna run? Based on current implementation, we'd have to manually run each service on ports 3000, 3001, 3002, etc.

I suggest just creating some implementation that takes care of this.

**Comment by rakhshankhan** [comment]
Another thing I was thinking is that we can use a form of layered architecture. See the following:

`sustain/
│
├── app/
│   ├── main.py                # Entry point to run the FastAPI app
│   ├── config.py              # Configuration and settings
│   ├── api/
│   │   ├── __init__.py
│   │   ├── v1/
│   │   │   ├── __init__.py
│   │   │   ├── endpoints/
│   │   │   │   ├── __init__.py
│   │   │   │   ├── example.py # Consider these your "controller" files that will reference your service logic.
│   │   ├── dependencies.py    # Dependency management in the case we have any.
│   ├── core/
│   │   ├── __init__.py
│   │   ├── config.py          # Application-level configurations (e.g., CORS, logging)
│   │   ├── security.py        # Authentication, authorization or other security details
│   ├── models/
│   │   ├── __init__.py
│   │   ├── example_model.py   # Database models or data schemas for Mongo
│   ├── services/
│   │   ├── __init__.py
│   │   ├── example_service.py # Application services/logic. This holds all logic which will be referenced by the endpoints.
│   ├── schemas/
│   │   ├── __init__.py
│   │   ├── example_schema.py  # Response/request object interfaces. Really important, we can discuss this further.
│   └── utils/
│       ├── __init__.py
│       └── helpers.py         # Helper functions if we need any.
└── requirements.txt           # Project dependencies
`

Using this architecture is going to keep everything organized cuz this could become a really big service. Also using request/response schemas, we can easily create a streamlined system between our request/responses from the model, main service, and this data-scraping one, which will make handling all the data a lot easier.

Lmk what u think

**Review by zaidokal — APPROVED** [review]

File: app/core/config.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from pydantic_settings import BaseSettings
+
+class Settings(BaseSettings):
+    APP_NAME: str = "SustAIn Data Scraper"
+    VERSION: str = "1.0.0"
+    API_PREFIX: str = "/api/scraper"
+    # Will add more setting (e.g., DATABASE_URL)
```
Comment (rakhshankhan) [inline app/core/config.py:7]: Don't think we need this for the data scraper?

Comment (skhan943) [reply app/core/config.py]: Fixed

**Review by rakhshankhan — APPROVED** [review]

**Review by arhamansarii — APPROVED** [review]

**Pull request merged**